uv.lock

```diff
@@ -267,7 +267,7 @@ name = "click"
 version = "8.1.8"
 source = { registry = "https://pypi.org/simple" }
 dependencies = [
-    { name = "colorama", marker = "sys_platform == 'win32'" },
+    { name = "colorama", marker = "platform_system == 'Windows'" },
```

Comment:
please update `uv` version locally to avoid this diff

Reply:
ACK thx will fix!